src/extension/background-script/actions/lnurl/auth.ts

```diff
@@ -94,11 +95,23 @@ export async function authFunction({
         authResponse?.data?.reason || "Auth: Something went wrong"
       );
     } else {
-      PubSub.publish(`lnurl.auth.success`, {
-        authResponse,
-        lnurlDetails,
-        origin,
-      });
+      // TODO: should use `AuthNotificationData`-type to align with usage in `notifications.ts`
```

Comment:
The types for this response are slightly off. We didn't notice this before but it should be tackled. Would do this in a follow-up PR

Reply:
All good